kitchen.dokken.yml

```diff
@@ -1,85 +1,66 @@
 driver:
   name: dokken
-  privileged: true # because Docker and SystemD/Upstart
-  chef_version: <%= ENV['CHEF_VERSION'] || 'stable' %>
+  privileged: true  # because Docker and SystemD
+  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
+  env: [CHEF_LICENSE=accept]
+  # Allow access to the Jenkins webui which is useful when troubleshooting
+  ports:
+    - 8080:8080
```

Comment:
I wonder if these should be commented out so people can uncomment if they need to. This would restrict users to only converge 1 instance locally at a time?

Reply:
I made this change in 17af8d6.

Feel free to tweak the language if desired.